docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# How to contribute
+
+Thanks for helping out!
+
+## Submitting changes
+
+Please submit changes through [pull requests](http://help.github.com/pull-requests/). When you send a pull request, the description will be pre-populated with a sample description. Please adhere to it as much as possible.
+
+Below is a list of other guidelines we try to follow for PRs.
+
+- Draft PRs should be ignored by other developers (unless explicitly stated by the PR author).
+- The PR author decides when a PR is ready for review.
+  - Marking a PR as “ready for review” implies that it can (most likely will) be merged after an approval (unless explicitly stated otherwise by the PR author in the PR description).
+  - When there has already been one review/reviewer on a PR, then that same reviewer will by default be expected to perform subsequent reviews on the same PR.
+- PRs should have only one reviewer unless extra reviewer(s) are explicitly requested by a PR author or reviewer (i.e., When a reviewer or author does not feel confident about a set of changes).
+- The PR author merges the PR into the target branch once the PR has been approved, but the reviewer can also decide to merge it immediately after an approval.
+  - If the author wishes for the PR not to be merged after review, this must be explicitly stated in the PR description.
+- PRs must be up to date with their target branch through the means of a rebase before being merged.
+- Only the PR author can make changes to their PR/branch. If a reviewer wants changes to be made, they request changes via a review.
+- For substantial changes, we prefer creating a "parent/feature" PR which receives multiple smaller PRs. For the parent PR and all the smaller child PRs, all the general guidelines apply.
```

Comment:
Can we mention how to do "feature" branching (ie. `feat.` etc. to trigger CI).

Reply:
We could reference: https://github.com/global-121/121-platform?tab=readme-ov-file#committing-and-creating-a-pull-request-pr

Or reword that section of the README entirely...

Reply:
@elwinschmitz good catch. I've moved that section into this file.

@diderikvw you're right, I also think it's not the best naming convention. Me, @elwinschmitz, and @RubenGeo were also discussing whether it makes sense to have it in the first place.

So, accordingly, I've also pushed a change where the workflows will run on all PRs, regardless of the naming convention, thus removing the current naming convention. We can, at a later stage, discuss and implement a naming convention if we need one, but I don't think there is a pressing problem that needs to be solved with a branch naming convention right now.